src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,7 +63,9 @@ public RobotContainer() {
   }
 
   private void setDefaultCommands() {
-    SwerveDriveSystem.setDefaultCommand(new DriveConstantControlCommand(SwerveDriveSystem, primaryController));
+    // SwerveDriveSystem.setDefaultCommand(new DriveConstantControlCommand(SwerveDriveSystem, primaryController));
```

Comment:
We should remove this change so it doesn't cause a merge conflict with the chassis code.